### DISCUSSION TITLE
Expedited review for critical defect in extension update?

### DISCUSSION THREAD

**Discussion by applied-kevinsalot** [microsoft/MicrosoftEdge-Extensions#90] [Q&A]
Hi,
Is there a process to request an expedited review for an update to an existing Edge extension? The latest version of our extension had a misconfiguration in the manifest and we need to quickly turn around and make another update to correct it. Our customers are unable to use it in the meantime.

Is there a process to go through for this?

Thank you,

**Comment by sivsouvamMSFT** [comment]
Hi @applied-kevinsalot 

Could you please share more details [here](mailto:ext_dev_support@microsoft.com).

**Reply by applied-kevinsalot** [reply]
Thanks for the quick reply. I just submitted more details to that address.

**Comment by sivsouvamMSFT** [comment]
[like]  Siv Souvam Prasad Bachu (Ushta... reacted to your message:
________________________________
